src/functionalTest/resources/1111006.metadata.json

```diff
@@ -1,6 +1,6 @@
 {
   "case_number": "1111222233334446",
-  "po_box": "TESTPO",
+  "po_box": "BULKSCANPO",
```

Comment:
PO Box is a number if you are updating it why not use a number ?
This applies to all the changes done below but hopefully a quick one

Reply:
Sure. But it is a string in Envelope model and all test data is in this format

Reply:
I think that's incorrect it has to be ` "po_box": "12626"` I didn't realised that the data type is incorrect in envelope model + DB. Will create a ticket and then we can update it as part of that ticket.